docroot/sites/now.uiowa.edu/modules/now_core/now_core.module

```diff
@@ -41,6 +42,16 @@ function now_core_form_alter(&$form, FormStateInterface $form_state, $form_id) {
   }
 }
 
+/**
+ * Implements hook_form_views_exposed_form_alter().
+ */
+function now_core_form_views_exposed_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  $view = $form_state->getStorage('view');
+  if ($form_id == 'views_exposed_form' && in_array($view['view']->id(), ['iowa_now_experts'])) {
+    $form_state->setErrorByName('field_person_research_areas_target_id', t('There are no matching results.'));
```

Comment:
I think this is marking the form as errored with any input, even if there are matching results. Instead, can we check if there's an error (and probably check what kind of error it is, too), and if it's there, alter the displayed message?

Reply:
Thanks @cory-skeers!